data_format/cellml.rst

```diff
@@ -0,0 +1,14 @@
+﻿.. image:: CellML_Logo_200x100.png
+   :align: right
+   :target: https://www.cellml.org/
+
+CellML in OpenCMISS
+===================
+
+OpenCMISS supports reading and solving models described in the `CellML language <https://www.cellml.org/>`_, an open standard for representing mathematical models, based on the XML Markup Language.
+
+A CellML model defines scalar *variables* and mathematical expressions on them (using a subset of `MathML 2.0 Content Markup <https://www.w3.org/TR/MathML2/>`_) within *components*, effectively sub-models with their own namespace for variables. *Imports* allow parts of other CellML models to be instantiated, permitting modular model construction and reuse, and *connections* tie together variables from different parts. Units are strictly enforced on variables, connections and maths in CellML. There are also extensive specifications for linking metadata to CellML model constructs. For in-depth documentation and specifications refer to `cellml.org <https://www.cellml.org/>`_.
```

Comment:
I'm not convinced this information needs to be duplicated here in addition to what is readily available on cellml.org. So I'd suggest removing this paragraph.